apps/rush-lib/src/index.ts

```diff
@@ -47,7 +47,7 @@ export {
 
 export {
   VersionMismatchFinder
-} from './data/VersionMismatchFinder'
+} from './data/VersionMismatchFinder';
```

Comment:
Some of the rules missed edge cases. They're more correct now.

---
In reply to: [129394309](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129394309) [](ancestors = 129394309)